react-native-capture.podspec

```diff
@@ -19,7 +19,7 @@ Pod::Spec.new do |s|
   s.source_files = "ios/**/*.{h,c,m,swift}"
   s.requires_arc = true
 
-  s.dependency "React"
+  s.dependency "React-Core"
```

Comment:
This is the correct dependency -- using "React" would break when building with frameworks in cocoa pods.